src/elements/sidebar/sidebar/sidebar.ts

```diff
@@ -0,0 +1,385 @@
+    }
+
+    this.opened = false;
+
+    // If the animation duration is zero, the animationend event is not always fired reliably.
+    // In this case we directly set the `opened` state.
+    if (!this.hasUpdated || isZeroAnimationDuration) {
+      this._handleClosing();
+    }
+  }
+
+  private _handleClosing(): void {
+    this.state = 'closed';
+    // We have to ensure that removing the animation skip instruction is done a tick later.
+    // Otherwise, it's removed too early and it doesn't have any effect.
+    setTimeout(() => this.toggleAttribute('data-skip-animation', false));
+    this.unTrap();
+
+    if (!isServer && (this.contains(document.activeElement) || this._isModeOver())) {
+      if (this._lastFocusedElement) {
+        setModalityOnNextFocus(this._lastFocusedElement);
+        this._lastFocusedElement?.focus();
+      } else {
+        // We don't know where to set the focus, but have to remove it, so we call blur
+        (document.activeElement as HTMLElement).blur();
+      }
+    }
+    this._lastFocusedElement = null;
+
+    this.stopAnimation();
+    this.didClose.emit();
+  }
+
+  private _takeFocus(): void {
+    // We prevent calling the focus stuff when not needed
+    if (this._inertController.isInert() || !this.isConnected) {
+      return;
+    }
+    const isModeOver = this._isModeOver();
+
+    if (isModeOver) {
+      this._inertController.activate();
+      this._escapableOverlayController.connect();
+    }
+
+    if (isModeOver || this.focusOnOpen) {
+      const firstFocusable = getFirstFocusableElement(
+        Array.from(this.children).filter((e): e is HTMLElement => e instanceof window.HTMLElement),
+      );
+      setModalityOnNextFocus(firstFocusable);
+      firstFocusable?.focus();
+    }
+
+    if (isModeOver) {
+      this._focusHandler.trap(this);
+      this._attachWindowEvents();
+    }
+  }
+
+  private _attachWindowEvents(): void {
+    this._windowEventsController?.abort();
+    this._windowEventsController = new AbortController();
+
+    // Close sidebar on backdrop click
+    window.addEventListener('pointerdown', this._pointerDownListener, {
+      signal: this._windowEventsController.signal,
+    });
+    window.addEventListener('pointerup', this._closeOnBackdropClick, {
+      signal: this._windowEventsController.signal,
+    });
+  }
+
+  // Check whether the pointerdown event target is triggered on the sidebar.
+  private _pointerDownListener = (event: PointerEvent): void => {
+    this._isPointerDownEventOnSidebar = isEventOnElement(
+      this.shadowRoot?.firstElementChild as HTMLDivElement,
+      event,
+    );
+  };
+
+  // Close sidebar on backdrop click.
+  private _closeOnBackdropClick = (event: PointerEvent): void => {
+    if (
+      !this._isPointerDownEventOnSidebar &&
+      !isEventOnElement(this.shadowRoot?.firstElementChild as HTMLDivElement, event)
+    ) {
+      this.close();
+    }
+  };
+
+  /** @internal */
+  public unTrap(): void {
```

Comment:
Option: `cedeFocus`
As dicsussed; It would be possible to reduce it to protected with calling `sidebar['cedeFocus']()`.